### ISSUE TITLE
Typo in OPF formulation in README

### ISSUE THREAD

**Issue by priyald17** [power-grid-lib/pglib-opf#43]
Equation (5) currently contains the term $-\mathbf{Y_i}^s|V_i|^2$.

However, I believe this should be $-(\mathbf{Y_i}^s)^*|V_i|^2$ -- that is, the shunt admittance should be conjugated.

**Comment by ccoffrin** [comment]
That's right, good catch.  The software is correct but this documentation is not.
